core/src/services/cloudService.ts

```diff
@@ -37,4 +37,13 @@ export interface CloudService {
    * @param invokeOptions parameters needed to call the invoke
    */
   invoke<T>(invokeOptions: InvokeRequest): Promise<T>;
+
+  /**
+   * @deprecated since 0.1.1-29 version
+   * Invoke a deployed Serverless function
+   * @param name Name of function to forget
+   * @param fireAndForget Don't listen for response if true
+   * @param payload Payload to send in invocation
+   */
+  invoke<T>(name: string, fireAndForget: boolean, payload?: any, headers?: StringParams): Promise<T>;
```

Comment:
I think we are missing the `pathParams` parameter

Reply:
I only added the previous method version, which didn't the pathParams in the CloudService invoke method. On the other hand, I will add the headers information in the JSDoc.